rs/cli/src/unit_tests/ctx_init.rs

```diff
@@ -1,4 +1,4 @@
-use std::{path::PathBuf, str::FromStr};
+use std::{path::PathBuf, process::Command, str::FromStr};
```

Comment:
Do we plan to use this code for non-unit tests as well?
If so, it would make sense to rename the directory from `unit_tests` to just plain `tests` since it's shorter and does not carry less information.

Reply:
I don't know. For the moment these are not even "unit tests", most are, but some do integrate with real services (Like governance canister). So what is your suggestion?

Reply:
How about renaming to `tests` then?